index.js

```diff
@@ -93,6 +98,12 @@ class I18n {
         // Handle one,other translation structure
         result = getOneOther(result, data);
 
+        if (EMPTY_VALUES.includes(result) && typeof this[EMPTY] === 'function') {
+            return this[EMPTY](
+                `${key}`, `${result}`, this.$scope, this.translations
```

Comment:
```suggestion
                `${key}`, result, this.$scope, this.translations
```

I think stringifying the empty value might be a little confusing here.

Reply:
Agreed